src/client/dfs/obj.c

```diff
@@ -674,7 +676,7 @@ dfs_obj_global2local(dfs_t *dfs, int flags, d_iov_t glob, dfs_obj_t **_obj)
 
 	oid_cp(&obj->oid, obj_glob->oid);
 	oid_cp(&obj->parent_oid, obj_glob->parent_oid);
-	strncpy(obj->name, obj_glob->name, DFS_MAX_NAME + 1);
+	memcpy(obj->name, obj_glob->name, DFS_MAX_NAME);
```

Comment:
.

Reply:
- [ ] Restore, add comments and pragma on `strncpy()` usage